src/plugins/home/server/services/sample_data/routes/install.ts

```diff
@@ -143,7 +143,15 @@ export function createInstallRoute(
 
       let createResults;
       try {
-        createResults = await context.core.savedObjects.client.bulkCreate(
+        const { getClient, typeRegistry } = context.core.savedObjects;
+
+        const includedHiddenTypes = sampleDataset.savedObjects
+          .map((object) => object.type)
+          .filter((supportedType) => typeRegistry.isHidden(supportedType));
```

Comment:
allow creating objects even if their type is hidden. We dont care if the object is importable or not in this case.

Reply:
This is just to future-proof the sample data so that it could include hidden objects right?

It doesn't have to be part of this PR, but I wonder if we shouldn't rather use the import API and store these saved objects as NDJSON instead of inside typescript files. It feels like it will also be more maintainable because we could just import/edit/export to update the sample data instead of editing source code.

Reply:
> This is just to future-proof the sample data so that it could include hidden objects right?

Yes. i was tesing my changes via sample data and I faced this issue. It can easily go unnoticed if someone changes their SO config without checking this part of the code.